static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -4,6 +4,11 @@ import _ from 'lodash';
 import { SearchkitComponent } from 'searchkit';
 import Icon from 'react-mdl/lib/Icon';
 
+const FILTER_ID_ADJUST = {
+  "birth_location": "profile.birth_country4",
+  "courses": "program.enrollments.title3"
```

Comment:
I reviewed the PR which added the original and I couldn't find a simple way around this unfortunately. I think the different Searchkit filters pass different ids and fixing this properly would require modifying the Searchkit classes to do this properly